packages/side-nav/src/CollapseToggle/CollapseToggle.tsx

```diff
@@ -33,10 +33,10 @@ export function CollapseToggle({
 
   return (
     <Tooltip
-      usePortal={false}
-      align="right"
-      justify="middle"
+      align={Align.Right}
+      justify={Justify.Middle}
       open={typeof hideTooltip === 'boolean' ? !hideTooltip : undefined}
+      renderMode={RenderMode.TopLayer}
```

Comment:
In the `In Layout` story, when toggling the toggle with the enter key, the tooltip sometimes appears in the wrong position and then returns to the correct spot after a second.

Reply:
I can't replicate this anymore. I'll mark it as resolved!